enhancements/network/bgp-ovn-kubernetes.md

```diff
@@ -197,6 +205,18 @@ or more nodes as route reflectors (https://issues.redhat.com/browse/CNF-10719).
 to connect to the route reflectors and do not have to maintain a full mesh with all other nodes in the cluster. Therefore,
 support for iBGP in this enhancement will be restricted to full mesh only.
 
+#### BFD
+
+Bi-directional Forwarding Detection (BFD) is used to detect link outages within milliseconds. This technology can be
+associated with routing in order to signal to a router that a peer has lost connection faster than the routing protocol
+can detect it. This assists with purging routes that are no longer viable, and recalculating optimal paths in the
+Routing Information Base (RIB).
+
+In OVN, BFD is supported today with the Multiple External Gateways (MEG) feature. However, there is no support for
+OVN to communicate with the FRR stack. Therefore for BFD support with BGP, FRR BFD daemon will be used in order to
+detect BFD link failures. It will then signal to FRR that the peer is down, and FRR will remove the routes routes from
```

Comment:
nit: this also serves the external router to remove routes advertised by OVNK / FRR-K8s

Reply:
clarified